frontend/public/components/dashboard/events-card/events-body.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { EventStream, EventStreamList, EventStreamChildProps } from '../../utils/events-stream';
```

Comment:
Nit: I'd use `event-stream` instead of `events-stream` (singular).

Reply:
fixed